lib/puppet/functions/generate_fernet_key.rb

```diff
@@ -0,0 +1,12 @@
+require 'securerandom'
+
+Puppet::Functions.create_function(:generate_fernet_key) do
```

Comment:
This should be namespaced to pulpcore
```suggestion
Puppet::Functions.create_function(:'pulpcore::generate_fernet_key') do
```

Reply:
:+1: